### PR TITLE
Add support for travis builds.

### PR DESCRIPTION
**Pull request by barrachri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +33,6 @@ Running the tests
 =============
 
 ```
-pip install -r test-requirements.txt
```
Comment (hjwp) [inline README.md]: if there's just one file let's just call it "requirements.txt".  anything else is a bit overcomplicated/yagni.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+sudo: false
+language: python
+stages:
+- name: test
+- name: deploy
+  if: tag IS present
+python:
+- '3.5'
```
Comment (hjwp) [inline .travis.yml]: are we using 3.5 anywhere?

Comment (barrachri) [reply .travis.yml]: No, I don't think so 👍

File: README.md (hunk 1 above)
Comment (barrachri) [reply README.md]: Right, fixed!

Comment (barrachri) [reply README.md]: More thoughts.... that's just for running the tests.

Using `ci-requirements.txt` I think make it even more explicit.

Comment (hjwp) [reply README.md]: well, it's required if you want to run the tests manually as well.  and it's required to do local dev.  just call it requirements.txt!   why make life complicated?

File: .travis.yml (hunk 2 above)
Comment (hjwp) [reply .travis.yml]: shall we drop 3.5 then?

**Comment by hjwp** [comment]
LGTM

**Pull request merged**